Dockerfile

```diff
@@ -72,68 +76,129 @@ FROM builder-base AS pcov-builder
 RUN pecl install pcov && docker-php-ext-enable pcov
 
 
-# Stage that builds all other extensions
-FROM builder-base AS extensions-builder
-RUN docker-php-ext-install bcmath intl zip pcntl bz2
+# Stage that builds bcmath
+FROM builder-base AS bcmath-builder
+RUN docker-php-ext-install bcmath
+
+
+# Stage that builds intl
+FROM builder-base AS intl-builder
+RUN docker-php-ext-install intl
+
+
+# Stage that builds zip
+FROM builder-base AS zip-builder
+RUN docker-php-ext-install zip
+
+
+# Stage that builds pcntl
+FROM builder-base AS pcntl-builder
+RUN docker-php-ext-install pcntl
+
+
+# Stage that builds bz2
+FROM builder-base AS bz2-builder
+RUN docker-php-ext-install bz2
+
+
+# Stage with PHP extensions, all validated
+FROM php-base AS all-extensions
```

Comment:
The point of this is that building them all in different stages is good for invalidation, but we want one (validated) place to copy them FROM into the final image so that the final has a single layer for this